### PR TITLE
Input test

### PR DESCRIPTION
**Pull request by gabimartini**
trello issue: https://trello.com/c/gc6ahudD

### CODE REVIEW

File: spec/features/managing_employees_spec.rb [hunk 1]
```diff
@@ -65,10 +71,46 @@
...
           page.select job_title, from: :employee_job_title
+          [
+            "Designated Behaviour Management Lead",
+            "Designated Safeguarding Lead",
+            "Designated SENCO",
+            "First Aider",
+            "Ofsted Registered Contact",
+            "Practice Leader"    
+          ].each do |r| 
+            if r ==  "Designated Safeguarding Lead" || r == "First Aider"
```
Comment (em-cd) [inline spec/features/managing_employees_spec.rb:83]: Do we need this loop if we are just checking two roles?

File: spec/features/managing_employees_spec.rb [hunk 2]
```diff
@@ -80,6 +122,35 @@
...
+        scenario 'it check the inputs' do
+          click_link 'Change'
+          expect(page).to have_content 'Edit an employee'
+          expect(page).to have_selector("input[value=#{surname}]")
+          expect(page).to have_selector("input[value=#{forenames}]")
+          expect(find_field('Home address').value).to eq address
+          expect(find_field('Home postcode').value).to eq postcode
+          expect(find_field('Date of birth').value).to eq dob.strftime
+          expect(find_field('employee_employed_from').value).to eq employed_from.strftime
+          page.has_select?(:employee_service_id, selected: service_name)
```
Comment (em-cd) [inline spec/features/managing_employees_spec.rb:134]: Looks like this is missing an `expect` - out of curiosity, if you changed this so the test should fail, e.g. by doing...

```
page.has_select?(:employee_service_id, selected: 'definitely not a service name or id')
```

...does the test still pass?

File: spec/features/managing_employees_spec.rb [hunk 3]
```diff
@@ -80,6 +122,35 @@
           expect(page).to have_content service_name
         end
 
+        scenario 'it check the inputs' do
+          click_link 'Change'
+          expect(page).to have_content 'Edit an employee'
+          expect(page).to have_selector("input[value=#{surname}]")
```
Comment (em-cd) [inline spec/features/managing_employees_spec.rb:128]: You can make this a bit neater with the [`have_field` matcher](https://www.rubydoc.info/gems/capybara/Capybara/Node/Matchers#has_field%3F-instance_method) (`have_field` is an alias for `has_field?` so you can use them interchangeably):

```
expect(page).to have_field(:employee_surname, with: surname)
```

**Review by em-cd — CHANGES_REQUESTED** [review]
Nice work on this! I've added some suggestions inline, let me know if you have any questions 😄

File: spec/features/managing_employees_spec.rb (hunk 2 above)
Comment (gabimartini) [reply spec/features/managing_employees_spec.rb]: You were right, it was passing. I've fixed that one :D

File: spec/features/managing_employees_spec.rb [hunk 4]
```diff
@@ -65,10 +71,46 @@
...
+            "Practice Leader"    
+          ].each do |r| 
+            if r ==  "Designated Safeguarding Lead" || r == "First Aider"
+            check(r) 
+            end
+          end
+          fill_in :employee_employed_from, with: employed_from
           check :employee_currently_employed
-
+          check :employee_has_food_hygiene
```
[the remark this answers is not in the document]
Comment (gabimartini) [reply spec/features/managing_employees_spec.rb:95]: I made the change 😄

File: spec/features/managing_employees_spec.rb (hunk 1 above)
Comment (em-cd) [reply spec/features/managing_employees_spec.rb]: Yes, getting rid of that `if` definitely makes it less complicated to read! You could even simplify it further, as in the expectations you're only checking one of them ("Designated Safeguarding Lead") so you could get rid of the `map` and just check the one field.

Or if you wanted to test that saving two roles works, you could just have two lines with `check`s and that would also be easier to read than the `map`.

Comment (gabimartini) [reply spec/features/managing_employees_spec.rb]: Ahh ok! I made the change, and add few checks more

**Review by em-cd — APPROVED** [review]
Looks great!

**Pull request merged**